### ISSUE TITLE
Anchor span highlight: use less saturated colours

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#2323]
Now the colour of the highlight is too dark, use a lighter, more pastel palette.

Especially the T highlight should be _very_ light, e.g. very light blue (blue is T colour) or perhaps most likely, very light gray, because nearly everything will almost always be enclosed in a T anchor.

![obrazek](https://github.com/user-attachments/assets/9d801afe-fdae-44d4-be52-e34a9c3fde15)

**Comment by adammertel** [comment]
The dark yellow color is used only for active anchors (anchor of the T that is already opened).
All other anchors are colored based on the entity class now. 
Feel free to reopen if you think that the issue was to addressed.

**Comment by davidzbiral** [comment]
Yes, I know, but you are all the time of course working in some subT, so you only see the very dark yello background _all the time_, which is not what we want. It is too saturated. Make it lighter, more pastel colours. I reopen.

**Comment by adammertel** [comment]
This is a fair point, I would try to find a better solution here

**Comment by adammertel** [comment]
I think this issue is solved by the new focus highlighting mode
![image](https://github.com/user-attachments/assets/f5130c32-57cb-4519-b851-653d74d9bf15)

**Comment by Ptrhnk** [comment]
@davidzbiral needs your review and confirmation

**Comment by davidzbiral** [comment]
@Ptrhnk Pls make the gray background lighter gray, otherwise this is a good solution - then close.
